diff -ur pwnsjp-orig/config.c pwnsjp-b04db2e255bebd652c262687afd49fd1ac685269/config.c
--- pwnsjp-orig/config.c	2022-02-26 11:00:27.000000000 +0200
+++ pwnsjp/config.c	2022-03-29 17:21:19.342728400 +0300
@@ -96,9 +96,10 @@
   if (strchr(config.filename, '/') == NULL)
   {
     bool hasext = strchr(config.filename, '.') != NULL;
-    int size = 1 + snprintf(NULL, 0, DICTDIR "/%s%s", config.filename, hasext ? "" : ".win");
+    char *dictdir = getenv("PWNSJP_DICTDIR"); if (!dictdir) dictdir = DICTDIR;
+    int size = 1 + snprintf(NULL, 0, "%s/%s%s", dictdir, config.filename, hasext ? "" : ".win");
     char *buffer = alloz(size, 1);
-    sprintf(buffer, DICTDIR "/%s%s", config.filename, hasext ? "" : ".win");
+    sprintf(buffer, "%s/%s%s", dictdir, config.filename, hasext ? "" : ".win");
     config.filename = buffer;
     atexit(deconfigure);
   }
