--- visual_regexp-3.1.tcl.font	2023-11-19 06:30:24.458841002 +0200
+++ visual_regexp-3.1.tcl	2023-11-19 06:30:24.460840942 +0200
@@ -1,7 +1,9 @@
+#!/usr/bin/env wish8.6
+#-*-tcl-*-
 package require starkit
 starkit::startup
 
-set version 3.0
+set version 3.1
 
 ###############################################################################################
 #
@@ -176,8 +178,8 @@
 				set sfont	{Courier 8};
 				set sbfont	{Courier 8 bold};
 			} else {
-				set sfont	6x13;
-				set sbfont	6x13bold;
+				set sfont	{Monospace 10};
+				set sbfont	{Monospace 10 bold};
 			}
 
 			set data(w:help)	[text .top.regexp.help \
@@ -756,7 +758,6 @@
 
 	pack .help.text -side top -fill both -expand 1 -padx 20
 
-	.help.text.t tag configure bold -font "[.help.text.t cget -font] bold"
 	.help.text.t insert 1.0 "Version:" bold " $::version
 
 " normal "Usage:" bold " tkregexp <sampleFile>
@@ -1437,7 +1438,7 @@
 		set output [text	$f.output \
 			-wrap						char \
 			-undo						1 \
-			-background				white 
+			-background				white \
 			-font						$::font_regexp \
 			-selectbackground		lightblue \
 			-selectborderwidth	0 \
