diff --git a/modules/FvwmPager/FvwmPager.c b/modules/FvwmPager/FvwmPager.c
index 7747e361..cbd7070e 100644
--- a/modules/FvwmPager/FvwmPager.c
+++ b/modules/FvwmPager/FvwmPager.c
@@ -736,7 +736,7 @@ void list_add(unsigned long *body)
 	while(t != NULL)
 	{
 		//t->m = newm;
-		fprintf(stderr, "WINDOW %s: mon: %s\n", t->window_name, t->m->name);
+		fvwm_debug(__func__, "WINDOW %s: mon: %s\n", t->window_name, t->m->name);
 		if (t->w == cfgpacket->w)
 		{
 			/* it's already there, do nothing */
diff --git a/modules/FvwmPager/x_pager.c b/modules/FvwmPager/x_pager.c
index fce74763..af797849 100644
--- a/modules/FvwmPager/x_pager.c
+++ b/modules/FvwmPager/x_pager.c
@@ -1905,7 +1905,7 @@ void SwitchToDesk(int Desk)
 
 	sprintf(command, "GotoDesk %s 0 %d", monitor_to_track ? m->name : "",
 			Desk + desk1);
-	fprintf(stderr, "%s: cmd: %s\n", __func__, command);
+        fvwm_debug(__func__, "cmd: %s\n", command);
 	SendText(fd,command,0);
 }
 
@@ -1931,7 +1931,7 @@ void SwitchToDeskAndPage(int Desk, XEvent *Event)
     mon->virtual_scr.Vy = vy * mon->virtual_scr.MyDisplayHeight;
     sprintf(command, "GotoDeskAndPage %s %d %d %d",
 		monitor_to_track ? mon->name : "", Desk + desk1, vx, vy);
-    fprintf(stderr, "%s: cmd: %s\n", __func__, command);
+    fvwm_debug(__func__, "cmd: %s\n", command);
     SendText(fd, command, 0);
 
   }
@@ -1959,7 +1959,7 @@ void SwitchToDeskAndPage(int Desk, XEvent *Event)
     if (y * mon->virtual_scr.MyDisplayHeight > mon->virtual_scr.VyMax)
       y = mon->virtual_scr.VyMax / mon->virtual_scr.MyDisplayHeight;
     sprintf(command, "GotoPage %s %d %d", monitor_to_track ? mon->name : "", x, y);
-    fprintf(stderr, "%s: cmd: %s\n", __func__, command);
+    fvwm_debug(__func__, "cmd: %s\n", command);
     SendText(fd, command, 0);
   }
   Wait = 1;
@@ -1974,7 +1974,7 @@ void IconSwitchPage(XEvent *Event)
 	  monitor_to_track ? mon->name : "",
 	  Event->xbutton.x * mon->virtual_scr.VWidth / (icon_w * mon->virtual_scr.MyDisplayWidth),
 	  Event->xbutton.y * mon->virtual_scr.VHeight / (icon_h * mon->virtual_scr.MyDisplayHeight));
-  fprintf(stderr, "%s: cmd: %s\n", __func__, command);
+  fvwm_debug(__func__, "cmd: %s\n", command);
   SendText(fd, command, 0);
   Wait = 1;
 }
