diff --git a/fvwm/bindings.c b/fvwm/bindings.c
index c7ea5b3..e0d5a06 100644
--- a/fvwm/bindings.c
+++ b/fvwm/bindings.c
@@ -466,33 +466,34 @@ static int ParseBinding(
 	{
 		action = SkipSpaces(action, NULL, 0);
 	}
+	if (action != NULL)
+	{
+		is_pass_through = is_pass_through_action(action);
+	}
 	if (
 		action == NULL || *action == 0 ||
 		(action[0] == '-' && !is_pass_through))
 	{
 		is_unbind_request = True;
 	}
-	else
+	else if (is_pass_through)
 	{
-		is_pass_through = is_pass_through_action(action);
-		if (is_pass_through)
+		/* pass-through actions indicate that the event be
+		 * allowed to pass through to the underlying window. */
+		if (window_name == NULL)
 		{
 			/* pass-through actions indicate that the event be
 			 * allowed to pass through to the underlying window. */
-			if (window_name == NULL)
+			/* It doesn't make sense to have a pass-through
+			 * action on global bindings. */
+			if (!is_silent)
 			{
-				/* It doesn't make sense to have a pass-through
-				 * action on global bindings. */
-				if (!is_silent)
-				{
-					fvwm_debug(
-						__func__,
-						"Invalid action for global "
-						"binding: %s", tline);
-				}
-
-				return 0;
+				fvwm_debug(
+					__func__,
+					"Invalid action for global "
+					"binding: %s", tline);
 			}
+			return 0;
 		}
 	}
 
